### ISSUE TITLE
What is the best way to create a new item with FeathersVuexFormWrapper

### ISSUE THREAD

**Issue by bzd2000** [feathersjs-ecosystem/feathers-vuex#381]
### Steps to reproduce

I want to create a new item with the FeathersVuexFormWrapper
Currently I use this code, but not sure if it is the right way
```
  <FeathersVuexFormWrapper
                        :item="getNewTimeEntry()"
                        watch
                      >
                        <template v-slot="{ clone, save, reset, remove }">
                          <q-form @submit="save">
                            <div class="row q-col-gutter-sm no-wrap">
                              <div class="col-2">
                                <q-input
                                  v-model="clone.date"
                                  dense
                                  input="date"
                                />
                              </div>
                              <div class="col-1">
                                <q-input
                                  v-model.number="clone.duration"
                                  dense
                                  input="number"
                                />
                              </div>
                              <div class="col">
                                <q-input dense v-model="clone.description" />
                              </div>
                              <div class="col-1">
                                <q-Btn
                                  icon="add"
                                  round
                                  size="xs"
                                  class="flex items-center"
                                  dense
                                />
                              </div>
                            </div>
                          </q-form>
                        </template>
                      </FeathersVuexFormWrapper>

 methods: {
    getNewTimeEntry: function() {
      const {TimeEntry} = this.$FeathersVuex.api;
      return new TimeEntry();
    }
  }

````

**Comment by marshallswain** [comment]
The steps are as follows:

1. Create a new item outside FeathersVuexFormWrapper
2. Pass the new item in as the `item` property.

```js
computed: {
   item() {
      const { TimeEntry } = this.$FeathersVuex.api
      const id = this.$route.params.id

      return id === 'new' ? new TimeEntry() : TimeEntry.getFromStore(id)
   },
   watch: {
      '$route.params.id': {
         handler(val) {
            const { TimeEntry } = this.$FeathersVuex.api
            const existingRecord = TimeEntry.getFromStore(val)
            if (!existingRecord) {
               TimeEntry.get(val)
            }
         },
         immediate: true
      }
}
```

**Comment by bzd2000** [comment]
ok thx!

**Comment by marshallswain** [comment]
I'm going to leave this open and add this information to the docs.  Thanks!

**Comment by marshallswain** [comment]
Closing now that I've created https://vuex.feathersjs.com/feathers-vuex-form-wrapper.html#example-usage-crud-form.

**Comment by bzd2000** [comment]
Thx for this update. 

I still of a question on this. It is more a javascript technical question. 

I'm working on a generic table based on quasar +feathers-vuex. 

In this table component I would like to add new instances etc. It is working fine, but one thing remains to make it more generic. 
It is the creation of a new instance dynamically wrt to the type of model. 

To create a new instance: 

const { Todo } = this . $FeathersVuex . api 
let todo = new Todo() 

Is there a way to set the type of model (Todo) based on a prop or something? 


thx, 
Filip. 



Van: "Marshall Thompson" <notifications@github.com> 
Aan: "feathersjs-ecosystem/feathers-vuex" <feathers-vuex@noreply.github.com> 
Cc: "filip" <filip.delperdange@telenet.be>, "State change" <state_change@noreply.github.com> 
Verzonden: Vrijdag 13 december 2019 06:21:54 
Onderwerp: Re: [feathersjs-ecosystem/feathers-vuex] What is the best way to create a new item with FeathersVuexFormWrapper (#381) 



Closing now that I've created [ https://vuex.feathersjs.com/feathers-vuex-form-wrapper.html#example-usage-crud-form | https://vuex.feathersjs.com/feathers-vuex-form-wrapper.html#example-usage-crud-form ] .

**Comment by bzd2000** [comment]
hello, 

Also this. 
It seems it possible to to save().then(), but it is not possible for remove and reset I think. 

kr, 
Filip. 


Van: "Marshall Thompson" <notifications@github.com> 
Aan: "feathersjs-ecosystem/feathers-vuex" <feathers-vuex@noreply.github.com> 
Cc: "filip" <filip.delperdange@telenet.be>, "State change" <state_change@noreply.github.com> 
Verzonden: Vrijdag 13 december 2019 06:21:54 
Onderwerp: Re: [feathersjs-ecosystem/feathers-vuex] What is the best way to create a new item with FeathersVuexFormWrapper (#381) 



Closing now that I've created [ https://vuex.feathersjs.com/feathers-vuex-form-wrapper.html#example-usage-crud-form | https://vuex.feathersjs.com/feathers-vuex-form-wrapper.html#example-usage-crud-form ] .
